mlflow/models/utils.py

```diff
@@ -264,3 +263,17 @@ def plot_lines(data_series, xlabel, ylabel, legend_loc=None, line_kwargs=None):
     ax.set(xlabel=xlabel, ylabel=ylabel)
 
     return fig, ax
+
+
+def create_wheeled_model(artifact_path, model_uri):
```

Comment:
Do we need this? Looks like it's just an alias to `WheeledModel.log_model`, I'd rather have only one top level API.

Reply:
I'd rather have this API and keep the other one internal. Adding a wheel to a model is a utility; I don't think we need to expose a new class with its own API surface to accomplish it

Reply:
Can we call this `add_libraries_to_model`? Referring to "wheeled models" creates a whole new "wheeled model" concept that users shouldn't need to reason about. Libraries instead of wheels gives us flexibility to add other input types in the future (eggs, etc)

Reply:
Yea, I added this as a utility since we agreed to avoid having the user learn a new concept. The alias makes it easier to just add wheels. 
We also give the user the ability to use the internal API if required.

For now I will only change this method name to `add_libraries_to_model`. The rest will remain as is.
@tomasatdatabricks  and @dbczumar do you have thoughts on any other name like:

- include_model_dependencies 
- include_model_libraries
- store_model_dependencies
- store_model_libraries
- download_model_dependencies
- download_model_libraries

Reply:
Or `install_x`